cardano-node/src/Cardano/Tracing/Kernel.hs

```diff
@@ -52,7 +54,8 @@ mapNodeKernelDataIO f (NodeKernelData ref) =
   readIORef ref >>= traverse f
 
 nkQueryLedger ::
-     (ExtLedgerState blk -> a)
+     IsLedger (LedgerState blk)
```

Comment:
This is because of https://github.com/input-output-hk/ouroboros-network/pull/2795